contracts/fast/FastTokenFacet.sol

```diff
@@ -272,6 +271,7 @@ contract FastTokenFacet is AFastFacet, IERC20, IERC1404 {
       external onlyDiamondFacet
       differentAddresses(p.from, p.to)
       onlyTokenHolder(p.from)
+      onlyExchangeActiveMember(p.from)
```

Comment:
This seemed to make sense, explicitly barring transfers _from_ a deactivated member.

Reply:
Yes!

Reply:
We probably will want to ask the question: when a member becomes de-activated:
- Can they still use allowances? Eg I am deactivated, have an allowance on your account - can I spend from your account?
- Can a member give allowance to a de-activated member?
Based on these answer, we might need to add `onlyExchangeActiveMember(p.spender)`